lib/game.rb

```diff
@@ -0,0 +1,72 @@
+require_relative "../config/environment.rb"
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  def current_player
+    board.cells.select{|cell| !cell.strip.empty?}.count.even? ? player_1 : player_2
```

Comment:
You can probably take advatange of `board#turn_count` here. 

`@board.turn_count.even? ? @player_1 : @player_2`